doc/run_experiment.rst

```diff
@@ -1075,7 +1075,7 @@ BayesianRidge
 DecisionTreeClassifier and DecisionTreeRegressor
     .. code-block:: python
 
-       {'max_features': ["auto", None]}
+       {'max_features': ["sqrt", None]}
```

Comment:
Why was this changed here?

Reply:
This was a deprecated option that I changed in the code but forgot to change it in the documentation so I included it here.